src/diart/models.py

```diff
@@ -45,14 +59,38 @@ def __call__(self, *args, **kwargs):
         return super().__call__(*args, **kwargs)
 
 
+class LazyWeSpeakerSpeakerEmbedding(WeSpeakerPretrainedSpeakerEmbedding, ABC):
```

Comment:
This class is not necessary, `LazyModel` should be able to take care of everything, we just need to make sure that `nn.Module` is replaced/removed.
`LazyModel` should be modified to not inherit from `nn.Module` and the `to()` method should be updated accordingly.